hoot-core/src/main/cpp/hoot/core/conflate/matching/MatchFactory.cpp

```diff
@@ -172,8 +172,7 @@ void MatchFactory::_tempFixDefaults()
   LOG_VARD(matchCreators);
   LOG_VARD(mergerCreators);
 
-  if ((matchCreators.size() == 0 || mergerCreators.size() == 0) &&
-      !ConfigOptions().getConflateEnableOldRoads())
+  if ((matchCreators.size() == 0 || mergerCreators.size() == 0))
   {
     throw HootException(
       "Empty match/merger creators only allowed when conflate.enable.old.roads is enabled.");
```

Comment:
I suggest you throw a different message in this exception.  Or maybe if they are empty, we could grab the defaults from `ConfigOptions::getMatchCreatorsDefaultValue()` and `ConfigOptions::getMergerCreatorsDefaultValue()` since this function is about fixing defaults.

Reply:
Yeah, the message is definitely wrong...missed thank.  Using defaults if they are empty is a good idea too.  will do that.